docs/html/reference/pip_install.rst

```diff
@@ -694,10 +694,21 @@ does not satisfy the ``--require-hashes`` demand that every package have a
 local hash.
 
 
+Local project installs
+++++++++++++++++++++++
+pip supports installing local project in both regular mode and editable mode.
+You can install local projects by specifying the project path to pip::
+
+$ pip install path/to/SomeProject
+
+During the installation, pip will copy the entire project directory to a temporary location and install from there.
```

Comment:
I would say `regular installation` just so people don't get the idea that it may apply to editable installs.